docs/units/standard_units.rst

```diff
@@ -48,6 +48,78 @@ all the existing predefined units of a given type::
     u            | 1.66054e-27 kg  | Da, Dalton                       ,
   ]
 
+
+Prefixes
+--------
+
+Most units can be used with prefixes, with both the standard SI prefixes and
+the IEEE 1514 binary prefixes (for ``bit`` and ``byte``) supported:
+
++------------------------------+
+|  Available decimal prefixes  |
++--------+-------------+-------+
+| Symbol |    Prefix   | Value |
```

Comment:
@larrybradley , you comment arrived just a little too late. Do you want to submit a follow-up PR for this?